src/pages/Assessments.js

```diff
@@ -9,20 +10,39 @@ export default function Assessments() {
       fname: 'Derek',
       lname: 'Chen',
       email: 'derek@nwplus.io',
-      score: 7,
+      score: {
```

Comment:
@dchen150 review the relevant changes for this schema in Table.js if u have time

Reply:
lgtm